app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: @post, url: new_post_path, method: :post, local: true, scope: :post do |f| %>
+  <%= f.hidden_field :author_id, value: params[:id] %><br>
+  <%= f.hidden_field :comments_counter, value: 0 %><br>
+  <%= f.hidden_field :likes_counter, value: 0 %><br>
+  <div class="flex-row col-flex">
+    <%= f.label :title %>
+    <%= f.text_field :title, size: "50x12", placeholder: "Write your title here..." %>
+    <%= f.label :Text %>
+    <%= f.text_area :text, size: "50x12", placeholder: "Write your post here..." %>
+    <%= f.submit "Submit" %>
+  </div>
+<% end %>
```

Comment:
- [ ] This form should have its route like `/users/:id/posts/new` it shouldn't be on the user posts page  you can add a button like `add a new post` then it takes you to the new form page

Reply:
Addressed ✔